tests/test_custom_models.py

```diff
@@ -3208,6 +3250,15 @@ def test_mixed_adapter_batches_lora_conv1d_emb(self):
         inputs = {"X": torch.arange(90).view(-1, 10).to(self.torch_device)}
         self.run_checks(peft_model, inputs)
 
+    def test_mixed_adapter_batches_lora_conv1d_emb_different_classifiers(self):
```

Comment:
Let's call this `test_mixed_adapter_batches_lora_conv1d_emb_multiple_modules_to_save`